pages_domain_test.go

```diff
@@ -0,0 +1,276 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+	"time"
+)
+
+const testPagesProjectName = "page-project"
+
+var testCreatedOn, _ = time.Parse(time.RFC3339, "2017-01-01T00:00:00Z")
+
+var testPagesDomain = PagesDomain{
+	ID:     "8232210c-6818-4e34-8d95-cc386874b8d2",
+	Name:   "washly.mvp",
```

Comment:
let's use documentation domains (ending in `.test`) or `example.com` instead of real domains.